### ISSUE TITLE
Docker configuration out of date?

### ISSUE THREAD

**Issue by anselanza** [HyperGAN/HyperGAN#224]
I am trying to get HyperGAN running via a Docker image (since a `Dockerfile` is provided), as I would much rather have a contained, pre-configured environment in which to use it (rather than fiddling with Python versions all over my machine).

Has anyone actually used the Docker setup successfully with this project? I notice that while the Dockerfile specifies a Python 3.4 environment, whereas the syntax in the project itself only works using 3.5. This also means that the Tensorflow version needs to be `-cp35` instead of `-cp34`. 

The `COPY` command also makes no sense, referring to a directories that do not (or no longer?) exist: `ADD build *.py shared /`. I changed this to a simple `COPY . .` to just copy everything.

Finally, the `ENTRYPOINT` also makes no sense: `ENTRYPOINT ["python3", "handler.py"]` - what is `handler.py`? This file also does not appear to exist anywhere.

I changed the entry point to `ENTRYPOINT ["python3", "hypergan", "test"]` but now I get the following error (from the Docker container): 
```
/usr/local/bin/python3: can't find '__main__' module in 'hypergan' 
```

Any idea where to go from here?

**Comment by mikkel** [comment]
Yes docker support is in a broken state at the moment.  I was trying to get it running on SONM to get more cheap compute.

**Comment by anselanza** [comment]
Any idea where to go from here, then? The hard part just seems to be getting files in the correct place?

**Comment by martyn** [comment]
The Dockerfile will not work until it is updated for 1.0. Will be after release.

**Comment by martyn** [comment]
The original diagnosis was correct: the old Dockerfile mixed Python/wheel versions and referenced missing paths and handler.py. The new foundation now has a tested python -m hypergan entrypoint, but a maintained container has not been implemented, so this issue remains open. The concrete deliverable is a pinned OCI profile built from the installed package, documented data/run mounts, working signal/exit handling, and a clean build-and-run smoke test; qualified GPU/cluster profiles follow their own gates. The [deployment plan](https://github.com/HyperGAN/HyperGAN/blob/develop/reports/resurrecting-hypergan-plan-2026-09-18.md) retains this work, and removing the old Dockerfile does not count as fixing it.

Audit record: [individual disposition and retained work](https://github.com/HyperGAN/HyperGAN/blob/f4053ae1/reports/issue-audit-2026-09-18.md), published in #302.
